Library/Homebrew/os/mac/version.rb

```diff
@@ -24,6 +24,18 @@ class Version < ::Version
         yosemite:    "10.10",
       }.freeze
 
+      # TODO: bump version when new macOS is released or announced
+      # and also update references in docs/Installation.md and
+      # https://github.com/Homebrew/install/blob/HEAD/install.sh
+      MACOS_NEWEST_UNSUPPORTED = "12.0"
+      private_constant :MACOS_NEWEST_UNSUPPORTED
+
+      # TODO: bump version when new macOS is released and also update
+      # references in docs/Installation.md and
+      # https://github.com/Homebrew/install/blob/HEAD/install.sh
+      MACOS_OLDEST_SUPPORTED = "10.14"
```

Comment:
```suggestion
      MACOS_OLDEST_SUPPORTED =  SYMBOLS[:mojave]
```
I guess it's better to show the version.

Reply:
I think it's fine as-was.